GoogleForCreators/web-stories-wp
# Template Updates: Cooking

## Context
Part of first round of template updates. updates recommended animations to current effects. Updates designs. New animation effects to be created before second round, then go over remaining unsupported animations and replace with thos.

## Summary
Updates recommended animations to current effects. Some design updates too. follows this doc:
https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/6970



https://user-images.githubusercontent.com/41136059/114252958-cf5dbd00-995c-11eb-9ac3-f90527166755.mp4


![Screen Shot 2021-04-09 at 5 56 22 PM](https://user-images.githubusercontent.com/41136059/114252975-ea303180-995c-11eb-8f0b-21de96a11fa0.png)




## Relevant Technical Choices
NA

## To-do


## User-facing changes
Updates cooking template with updates described in ticket:
https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/6970


## Testing Instructions
View cooking template and see that these changes took place:
https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/6970


### QA
Follow testing instructions above.





### UAT








## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6970
